src/redist/targets/DownloadPackage.csproj

```diff
@@ -14,4 +14,6 @@
 
   <!-- Workaround newer sdk preventing us from downloading Microsoft.AspNetCore.App via PackageReference above. -->
   <Target Name="UpdateAspNetToFrameworkReference" />
+  <!-- Workaround for: https://github.com/dotnet/sdk/issues/3104 -->
```

Comment:
I closed https://github.com/dotnet/sdk/issues/3104

You can set UseAppHost=false. This is ok too, but neither of these workarounds are around product bugs. This is just because the DownloadPackage.csproj is a big hack. We can switch to NuGet PackageDownload feature when we have a new enough nuget in stage0.